docs/docs/examples/guides/object_detection_example.md

```diff
@@ -0,0 +1,610 @@
+    - component: yolo_executor
+    - component: tf_to_signal
+components:
+  yolo_executor:
+    component: object_detection_components::YOLOExecutor
+    display_name: YOLO Executor
+    events:
+      transitions:
+        on_configure:
+          lifecycle:
+            component: yolo_executor
+            transition: activate
+        on_load:
+          lifecycle:
+            component: yolo_executor
+            transition: configure
+        on_activate:
+          load:
+            component: yolo_to_marker
+    parameters:
+      rate: !!float 1.0
+      conf_threshold: 0.4
+      model_file: /files/best.onnx
+      classes_file: /files/coco8.yaml
+    inputs:
+      rgb_image: /video_player_component/video_feed
+    outputs:
+      bounding_boxes: /yolo_executor/bounding_boxes
+  yolo_to_marker:
+    component: component_utils::YoloToMarker
+    display_name: YOLO to marker
+    events:
+      transitions:
+        on_load:
+          lifecycle:
+            component: yolo_to_marker
+            transition: configure
+        on_configure:
+          lifecycle:
+            component: yolo_to_marker
+            transition: activate
+        on_activate:
+          load:
+            component: signal_point_attractor
+    parameters:
+      rate: !!float 5.0
+      object_distance: 0.4
+    inputs:
+      json_input: /yolo_executor/bounding_boxes
+    outputs:
+      marker_pose: /yolo_to_marker/state_command
+  signal_point_attractor:
+    component: aica_core_components::motion::SignalPointAttractor
+    display_name: Signal Point Attractor
+    events:
+      transitions:
+        on_configure:
+          lifecycle:
+            component: signal_point_attractor
+            transition: activate
+        on_load:
+          lifecycle:
+            component: signal_point_attractor
+            transition: configure
+    parameters:
+      rate: !!float 2.0
+    inputs:
+      state: /hardware/robot_state_broadcaster/cartesian_state
+      attractor: /cartesian_transformation/output
+    outputs:
+      twist: /signal_point_attractor/twist
+  cartesian_transformation:
+    component: aica_core_components::signal::CartesianTransformation
+    display_name: Cartesian Transformation
+    inputs:
+      input_1: /tf_to_signal/pose
+      input_2: /yolo_to_marker/state_command
+    outputs:
+      output: /cartesian_transformation/output
+  tf_to_signal:
+    component: aica_core_components::ros::TfToSignal
+    display_name: TF to Signal
+    events:
+      transitions:
+        on_configure:
+          lifecycle:
+            component: tf_to_signal
+            transition: activate
+        on_load:
+          lifecycle:
+            component: tf_to_signal
+            transition: configure
+          load:
+            component: cartesian_transformation
+    parameters:
+      frame: camera_frame
+    outputs:
+      pose: /tf_to_signal/pose
+  video_player_component:
+    component: component_utils::VideoPlayer
```

Comment:
this is no longer valid and should be replaced with the Camera Streamer references. Technically that also warrants for a new screenshot of the app